docs/book/src/architecture/controllers/machine.md

```diff
@@ -1 +1,67 @@
-# Machine
+# Machine  Controller
+
+The machine controller is responsible for keeping the machine object in sync with the infrastructure machine object as
+as well as managing references and properly deleting dependent objects.
+
+Its main responsibilities are:
+
+* Setting an OwnerReferences on various objects, including adopting machines into clusters.
```

Comment:
This wording is a bit weird, since it seems weird to say a machine adopts itself into a cluster.